lighthouse.ts

```diff
@@ -1,11 +1,11 @@
-const spawnSync = require('child_process').spawnSync;
+import { spawnSync } from 'child_process';
+import sanitize from 'sanitize-filename';
+
 const lighthouseCli = require.resolve('lighthouse/lighthouse-cli');
```

Comment:
Out of curiosity, why is this one still `require`ed?

Reply:
There isn't currently an equivalent for `require.resolve` in node with modules. Eventually it will support [`import.meta.resolve`](https://nodejs.org/api/esm.html#esm_no_require_resolve) but it doesn't yet.

We are currently transpiling from ES Modules => CommonJS, but TS (which does that transformation) doesn't support transpiling `import.meta.resolve` to `require.resolve`.

Reply:
Interesting